src/error/specification

```diff
@@ -1266,7 +1266,7 @@ ingroup:plugin
 module:path
 
 number:202
-description:Could not change to user. You might want to try as root (use sudo !!).
+description:Could not change to user.
```

Comment:
Better to not mix unrelated changes into PRs.

Reply:
Why is this change unrelated? I created the error 202 and in order to reduce the number of errors I changed the text so I can reuse it for another error which is very similar but not worth distinguishable for the program (root -> user & user -> root changes are both the same problem but the old text above only includes root -> user)